src/ros_license_toolkit/package.py

```diff
@@ -97,6 +97,10 @@ def __init__(self, path: str, repo: Optional[Repo] = None):
         # this is Optional, because it is only evaluated on the first call
         self._license_tags: Optional[Dict[str, LicenseTag]] = None
 
+        # If the tag is wrong (like BSD) but the actual license can
+        # be found out through declaration, this field contains the tag
+        self.inofficial_license_tag: Dict[str, str] = {}
```

Comment:
Where is this used?
Would is the context? Something like `{"BSD": "BSD-3-Clase"}`? I think this would not be a good idea. Instead, we should save this in the license tag itself. I mean, the interpretation that is is in fact BSD-3-clause. And as we agreed only for two possible reasons: 
- The package has only one license tag and only one license text that we can identify as a spdx license, or
- Multiple license tags, each with and explicitly linked text, that we can identify.